x-pack/plugins/observability_onboarding/server/saved_objects/observability_onboarding_status.ts

```diff
@@ -46,6 +47,18 @@ export interface SavedObservabilityOnboardingFlow
   updatedAt: number;
 }
 
+const LogFilesStateSchema = schema.object({
```

Comment:
Thanks for adding this! I briefly looked through the code and I could not see that this schema is being used directly in your routes (is it?)